dotnet/src/webdriver/JavaScriptEngine.cs

```diff
@@ -380,6 +380,9 @@ private void OnScriptBindingCalled(object sender, BindingCalledEventArgs e)
             if (e.Name == MonitorBindingName)
             {
                 DomMutationData valueChangeData = JsonConvert.DeserializeObject<DomMutationData>(e.Payload);
+                var locator = By.CssSelector($"*[data-__webdriver_id='{valueChangeData.TargetId}']");
+                valueChangeData.Element = driver.FindElements(locator)[0];
```

Comment:
It throws exception in case of no elements were found. Should we be safe here?

Reply:
Cool, I'll use `FirstOrDefault()` then